src/translations/en_translations.json

```diff
@@ -129,6 +129,33 @@
+  "accessibilityQuestion": "How does the BHA plan to address and improve accessibility and belonging for users?\n",
+  "accessibilityAnswer": "The BHA plans to make changes such as training behavioral health professionals to respond respectfully to varying cultures, languages, and other considerations affecting marginalized populations. The BHA also plans to standardize guidelines across payer systems to make it easier for providers to enroll in public and private insurance networks and for patients to access. A final  example to address accessibility is to have an adequate, equitable, and complete continuum of behavioral health services, and address current disparities. \n\nFor user accessibility, the website is accessible to people with differing physical disabilities, such as blindness and low sight, deaf and hard of hearing, and mobile limitations restricting mouse movement. OwnPath strives to use clear, plain language, people-first language and transcultural translation to enhance the usability for all people of Colorado. The website is built for low-bandwidth scenarios and mobile-first usage. \n",
+  "feedbackQuestion": "Where can I give feedback on the tool?",
+  "feedbackAnswer": "Submit suggestions to this [form.](https://docs.google.com/forms/d/e/1FAIpQLSfXb19-4PgIamwjgg2J0vCp8nvCXNCYvz3v2tDtQFF0uNCTUA/viewform)",
+  "dataCollectionQuestion": "Will searches or any personal information be stored?",
+  "dataCollectionAnswer": "OwnPath will not capture any protected health information (PHI) on the site.\n",
+  "updatesQuestion": "What updates have been made?\n",
+  "updatesAnswer": "OwnPath officially launched on July 26, 2022 and will be continuously improved upon based on user and community feedback. Have an idea for improvement on this website? [Submit a suggestion via this Google form.](https://docs.google.com/forms/d/e/1FAIpQLSfXb19-4PgIamwjgg2J0vCp8nvCXNCYvz3v2tDtQFF0uNCTUA/viewform)\n",
+  "moreInfoQuestion": "Where can I find more information?",
+  "moreInfoAnswer": "[The BHA landing page](http://bit.ly/BHA-Colorado) includes additional information about OwnPath, including a project dashboard, an online form to share feedback and submit questions, toolkit resources, fact sheets, and more—in English and Spanish.",
+  "toolkitQuestion": "Where can I find OwnPath materials to print and share with my community?",
+  "toolkitAnswer": "We’ve created printable versions of select OwnPath documents, and will continue to add more as they become available. \n\n- Download the FAQ in: <a href='/resources/BHA OwnPath FAQs (EN).pdf' target='_blank' download>English</a>,  <a href='/resources/BHA OwnPath FAQs (ES).pdf' target='_blank' download>Spanish</a> \n\n- Download the OwnPath Overview one pager in:  <a href='/resources/BHA OwnPath one pager (EN).pdf' target='_blank' download>English</a>,  <a href='/resources/BHA OwnPath one pager (ES).pdf' target='_blank' download>Spanish</a>\n",
```

Comment:
feels a lil scary to have html in the translation strings, and it also feels like here we'd lose the user-friendliness of using markdown if the translator has to deal w html. i think it'd be safer to use the [react i18next trans component](https://react.i18next.com/latest/trans-component) for the last Q since it exists to handle these kinda cases

Reply:
I guess it doesn't give me bad vibes because it's not really "user supplied" input -- everything has to still be pulled into JSON and then reviewed and approved via PR process. I also don't agree that arbitrary <1> tags are more/less user friendly than html tags. And yes it exists to handle these cases, but not when we're trying to source the content directly from a non-technical document. 

all that being said, not strongly held opinions. My brain prefers the straight up HTML to the random number tags but I'm happy to go with the <Trans> component approach. I think it's nice to use the markdown without raw html when that's applicable though.. thoughts?